static/css/style.css

```diff
@@ -69,6 +69,14 @@ td {
     margin-bottom: 3px;
  }
 
+.file_metadata tr:not([class="description"]) {
+    display: none;
+}
+
+.file_metadata select option:not([value="3"]){
```

Comment:
Please add a comment here, to explain that "3" represents table of contents.  Using the numeric value is less fragile than using the display value "tableOfContents", but needs explanation.